tasks/broad/ImputationTasks.wdl

```diff
@@ -370,13 +371,28 @@ task UpdateHeader {
   String disable_sequence_dict_validation_flag = if disable_sequence_dictionary_validation then "--disable-sequence-dictionary-validation" else ""
 
   command <<<
-    ## update the header of the merged vcf
+    set -e -o pipefail
+
+    ## update the header of the merged vcf with the reference dictionary
     gatk --java-options "-Xms~{command_mem}m -Xmx~{max_heap}m" \
     UpdateVCFSequenceDictionary \
     --source-dictionary ~{ref_dict} \
     --output ~{basename}.vcf.gz \
     --replace -V ~{vcf} \
     ~{disable_sequence_dict_validation_flag}
+
+    ## update header with pipeline_header_line if provided
+    if [ -n "~{default="" pipeline_header_line}" ]; then
+      mv ~{basename}.vcf.gz temp.vcf.gz
+      mv ~{basename}.vcf.gz.tbi temp.vcf.gz.tbi
+
+      bcftools view -h --no-version temp.vcf.gz > header.txt
+      TOTAL_LINES=$(wc -l < "header.txt")
+      sed -i "${TOTAL_LINES}i\##~{pipeline_header_line}" header.txt
```

Comment:
any way we can check and make sure the user didn't already input a string that starts with `##`?

Reply:
yea just added the change, testing in a wdl now (tested in the docker container and it worked)

Reply:
https://app.terra.bio/#workspaces/allofus-drc-imputation/AOU_ANVIL_imputation_reference_panel_validation/submission_history/51d4c524-7b5a-430a-a2d3-89765ea5541a workflow witha value starting with "#" characters (and a # character in the middle ot make sure it didnt get removed)

<img width="1297" height="163" alt="Screenshot 2025-09-04 at 1 11 58 PM" src="https://github.com/user-attachments/assets/cd8b95f5-823c-46d2-81b1-0db74a882ece" />